### PR TITLE
add: new patterns to pollution

### PR DESCRIPTION
**Pull request by Thomzoy**
## Description

Provide additional pollution patterns, namely:

- webpage URLs and emails
- doctor list (coming from leftside margin note)
- biology table
- coding sections

## Checklist

- [x] If this PR is a bug fix, the bug is documented in the test suite.
- [x] Changes were documented in the changelog (pending section).
- [x] If necessary, changes were made to the documentation (eg new pipeline).

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 - Add attribute `section` to entities
 - Add new cases for separator pattern when components of the TNM score are separated by a forward slash
 - Add NER `eds.adicap` pipeline to identify ADICAP codes
+- Add pattenrs to `pollution` pipeline and simplifies activate/deactivate of specific patterns
```
Comment (Thomzoy) [inline changelog.md:13]:
```suggestion
- Add patterns to `pollution` pipeline and simplifies activate/deactivate of specific patterns
```

File: changelog.md [hunk 2]
```diff
@@ -10,6 +10,7 @@
 - Add attribute `section` to entities
 - Add new cases for separator pattern when components of the TNM score are separated by a forward slash
 - Add NER `eds.adicap` pipeline to identify ADICAP codes
+- Add patterns to `pollution` pipeline and simplifies activate/deactivate of specific patterns
```
Comment (percevalw) [inline changelog.md:13]:
```suggestion
- Add patterns to `pollution` pipeline and simplifies activating or deactivating specific patterns
```

File: changelog.md [hunk 3]
```diff
@@ -10,6 +10,7 @@
...
 
 ### Changed
```
Comment (percevalw) [inline changelog.md:15]: A word about the breaking change ?

```suggestion
### Changed
- Simplified the configuration scheme of the `pollution` pipeline
```

File: docs/pipelines/core/normalisation.md [hunk 4]
```diff
@@ -191,6 +191,48 @@ The two are independent: a matcher can use the `NORM` attribute but keep exclude
 
 ![Pollution alignment](./resources/alignment.svg)
 
+#### Types of pollution
+
+Pollution can come in various forms  in clinical texts. We provide a small set of possible pollution, from which you can pick whichever you want to activate.
+
+For instance, if you want to consider biology tables as pollution, you can instantiate the `normalizer` pipeline as follows:
```
Comment (percevalw) [inline docs/pipelines/core/normalisation.md:198]:
```suggestion
Pollution can come in various forms in clinical texts. We provide a small set of possible pollutions patterns that can be enabled or disabled as needed.

For instance, if we consider biology tables as pollution, we only need to instantiate the `normalizer` pipe as follows:
```

**Review by percevalw — APPROVED** [review]
Great thanks ! I've left a few comments regarding the documentation changes

**Pull request merged**